### DISCUSSION TITLE
Different command / sub-command syntax possible?

### DISCUSSION THREAD

**Discussion by jamsouf** [laravel-zero/laravel-zero#469] [General]
Let's say I want to define multiple commands under the same namespace, e.g.
`servers list`, `servers create` and `servers retrieve`.

The default way in Laravel (Zero) would be to have the following signatures:
`servers:list`, `servers:create` and `servers:retrieve`

I personally find it more elegant to use the commands in the console with `servers list` instead of `servers:list`, just like the Stripe CLI or the GitHub CLI for example do.

Is it possible to achieve such a syntax?

**Comment by owenvoke** [comment]
I don't think this is possible. Behind the scenes, Laravel and Laravel Zero use Symfony Console, and it doesn't seem to support this.
